packages/graphql-elasticsearch-transformer/src/resources.ts

```diff
@@ -11,6 +11,14 @@ import {
 } from 'graphql-mapping-template'
 import { toUpper, plurality, graphqlName, ResourceConstants, ModelResourceIDs } from 'graphql-transformer-common'
 
+interface Mappings {
```

Comment:
nit: the mapping interface in here and in transformer context might want to be the same in terms of naming convention though I think it's only best defined in the transformer context

Reply:
Made this change, interface is now MappingParameters and is imported from transformer context